fe/fe-core/src/main/java/com/starrocks/external/iceberg/cost/IcebergFileStats.java

```diff
@@ -195,8 +195,10 @@ public static Map<Integer, Object> toMap(Map<Integer, Type.PrimitiveType> idToTy
         }
         ImmutableMap.Builder<Integer, Object> map = ImmutableMap.builder();
         idToMetricMap.forEach((id, value) -> {
-            Type.PrimitiveType type = idToTypeMapping.get(id);
-            map.put(id, Conversions.fromByteBuffer(type, value));
+            if (idToTypeMapping.containsKey(id)) {
```

Comment:
The `containsKey` and `get` will query map repeatedly, so why not `get` firstly and then check the return value (whether is null) ?

Reply:
ok